Packs/Core/Playbooks/playbook-Uncommon_creation_or_access_operation_of_sensitive_shadow_copy_by_a_high_risk_process_README.md

```diff
@@ -0,0 +1,64 @@
+This playbook addresses the following alerts:
+ 
+- Uncommon creation or access operation of sensitive shadow copy by a high-risk process
+ 
+Playbook Stages:
+  
+Triage: 
+ 
+- Check if the causality process image (CGO) is signed or not
+ 
+Investigation:
+ 
+- Examine process details, prevalence, and historical data for similar alerts
+- If CGO is unsigned:
+  - Check the CGO process prevalence
+  - Check if the process image path is common
+- If CGO is signed:
+  - Check process image name
+  - Check initiating process image name
+  - Check if username is SYSTEM
+  - Check if host is a server
+  - Check for previous similar alerts
+ 
+Containment:
+ 
+- Terminate causality process (CGO) process - if CGO a malicious behavior or unsigned process from uncommon path attempted to create or access shadow copy
```

Comment:
if CGO a malicious behavior? i think you are missing a word here. if CGO unsigned? if CGO is what?

Reply:
It is still unclear when will the playbook terminate the CGO